proguard-custom.txt

```diff
@@ -110,3 +110,15 @@
 -dontwarn android.support.v4.**
 -dontwarn android.net.SSLCertificateSocketFactory
 -dontwarn android.net.http.*
+
+
+# These clases contain references to google maps
```

Comment:
Not sure about adding support for google maps. This is particular case and I think it could be added only in project who needs this setup, not for all.